src/audio/AudioPlayer.ts

```diff
@@ -338,9 +350,28 @@ export class AudioPlayer extends EventEmitter {
 	}
 }
 
+/**
+ * Stringifies an AudioPlayerState instance
+ *
+ * @param state The state to stringify
+ */
+function stringifyState(state: AudioPlayerState) {
+	return JSON.stringify({
+		...state,
+		resource: Boolean((state as any).resource),
+		stepTimeout: Boolean((state as any).stepTimeout)
```

Comment:
Those should be casted to the proper types, avoid casting to `any`.

Reply:
Addressed with `Reflect.has`